### PR TITLE
Text utilities and list

### PR DESCRIPTION
**Pull request by manujagobind**
**Summary**

This change adds the following:
- Styling for text utilities
- Documentation for text utilities
- Styling for lists component
- Documentation for lists component
- Updates index page with new components

**Third Party Software Used**

This change makes use of the following external code:
- Uses [FontAwesome v5.15](https://fontawesome.com/v5.15/icons) for icons

**View Live**
Please refer to the Netlify section below for the preview link.

### CODE REVIEW

File: src/pages/typography.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Gobind Manuja" />
+    <meta name="description" content="Pleasant UI - A CSS component library." />
+    <title>Cards | Pleasant UI</title>
```
Comment (arjunbharti) [inline src/pages/typography.html]: change title here

**Review by arjunbharti — APPROVED** [review]
Just a minor change, everything else looks good!

**Review by irajbhinde — COMMENTED** [review]
1) Are you using mobile-first approach? Because for desktop your right side of screen looks very empty
2) Instead of a HOME footer, you can add previous & next component footer
3) You can add one more type of list - spaced list, if time permits. 

Otherwise whatever you've written looks fine

**Comment by manujagobind** [comment]
> 1. Are you using mobile-first approach? Because for desktop your right side of screen looks very empty
> 2. Instead of a HOME footer, you can add previous & next component footer
> 3. You can add one more type of list - spaced list, if time permits.
> 
> Otherwise whatever you've written looks fine

Thanks @irajbhinde - Currently, I have my demo within a fixed width container hence you see a lot of free space. I'll fix this as well as the footer as part of documentation PR.

**Pull request merged**